lms/migrations/versions/fdaad19893bb_enable_ms_onedrive.py

```diff
@@ -0,0 +1,30 @@
+"""
+Enable MS OneDrive files in all installations created since Jan 1st, 2019.
+
+Revision ID: fdaad19893bb
+Revises: 2fd66b9ab1a1
+Create Date: 2021-11-12 13:36:11.755526
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "fdaad19893bb"
+down_revision = "2fd66b9ab1a1"
+
+
+def upgrade():
+    conn = op.get_bind()
+
+    conn.execute(
+        """
+        UPDATE application_instances
+            SET settings = settings || jsonb '{"microsoft_onedrive":{"files_enabled": true}}'
+        WHERE created >= '2019-01-01'::date
+            AND settings <> '{}' and developer_key is not null"""
+    )
```

Comment:
What will this to do application instances that have other settings, like `canvas.groups_enabled`, `canvas.sections_enabled`, `blackboard.files_enabled`, etc?